CHANGELOG.md

```diff
@@ -7,3 +7,21 @@ and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## 1.0.1
```

Comment:
This can be left out until we cut a release (using `yarn version ...` to update `package.json` and tag the commit). No need to fix for this PR though. Thanks for the `CHANGELOG` content!